tests/i18n/tests.py

```diff
@@ -1758,51 +1758,49 @@ def test_parse_literal_http_header(self):
         Now test that we parse a literal HTTP header correctly.
         """
         g = get_language_from_request
-        r = self.rf.get("/")
-        r.COOKIES = {}
```

Comment:
Tiny question: if these tests are resetting/forcing the cookies at the beginning, there probably was a reason, wasn't there?

Should we keep it, or are we confident that the `self.rf.get` is starting with empty cookies anyway (no leaking between the shared `self.rf` instance)?

Reply:
I think it was just copypasta from the tests that were deliberately setting cookies.

Only `Client` stores cookies from responses, so the cookies are always empty here, unless someone sets them on the `RequestFactory`